### PR TITLE
Add LICENSE file per project policy

### PR DESCRIPTION
**Pull request by kislyuk**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License
+
+Copyright (c) 2017 The Human Cell Atlas Consortium, https://humancellatlas.org
```
Comment (freeman-lab) [inline LICENSE]: let's make it `Copyright (c) 2017 Human Cell Atlas`

**Comment by freeman-lab** [comment]
just suggested the line to use for this and other projects

File: LICENSE (hunk 1 above)
Comment (ttung) [inline LICENSE]: Is there a requirement to slap copyright headers on all source files?

**Comment by kislyuk** [comment]
I would like to avoid any such requirement if possible. @freeman-lab?

**Comment by freeman-lab** [comment]
@kislyuk @ttung nope, doesn't need to be in all source files, just in the top level `LICENSE`

**Pull request merged**